app/install/aws-linux.md

```diff
@@ -76,6 +76,15 @@ baseurl=https://kong.bintray.com/kong-rpm/amazonlinux/amazonlinux
     should be performing migrations at a time. This limitation is lifted for
     Kong 0.15, 1.0, and above.
 
+    If you are going to run Kong in [DB-less mode](/{{site.data.kong_latest.release}}/db-less-and-declarative-config/),
+    you can skip the steps above. Instead, make sure you set the `database` option
+    to `off` and the `declarative_config` option to the path of your `kong.yml` file:
```

Comment:
The instructions should also mention that a good starting point is to create a new `kong.yml` file using 

```
kong config init
```

which produces a valid `kong.yml` in the current directory. It makes it _a lot_ easier to follow the step-by-step without having to dive into another tutorial.